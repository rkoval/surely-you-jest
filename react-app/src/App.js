import React, { Component } from 'react'
import logo from './jest.png'
import './App.css'

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h1 className="App-title">Surely, You Jest!</h1>
          <img src={logo} style={{ width: '80px', height: '80px' }} />
        </header>
        <p className="App-intro">
          A VERY SERIOUS TALK ON AUTOMATED TESTING (using Facebook&apos;s Jest)
        </p>
      </div>
    )
  }
}

export default App
