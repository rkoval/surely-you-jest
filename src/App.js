import React, { Component } from 'react'
import logo from './jest.png'
import './App.css'

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h1 className="App-title">Surely, You Jest!</h1>
          <img src={logo} style={{ width: '80px', height: '80px' }} />
        </header>
        <p className="App-intro">
          To get started, edit <code>src/App.js</code> and save to reload.
        </p>
      </div>
    )
  }
}

export default App
