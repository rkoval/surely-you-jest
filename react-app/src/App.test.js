import React from 'react'
import App from './App'
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const component = renderer.create(<App />)
  expect(component.toJSON()).toMatchSnapshot()
})
