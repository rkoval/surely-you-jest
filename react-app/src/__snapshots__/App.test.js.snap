// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
      Surely, You Jest!
    </h1>
    <img
      src="jest.png"
      style={
        Object {
          "height": "80px",
          "width": "80px",
        }
      }
    />
  </header>
  <p
    className="App-intro"
  >
    A VERY SERIOUS TALK ON AUTOMATED TESTING (using Facebook's Jest)
  </p>
</div>
`;
